client/components/UserHome.jsx

```diff
@@ -1,23 +1,63 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import {connect} from 'react-redux';
+import React, {Component} from 'react';
+import Select from 'react-select';
+import axios from 'axios';
 
-export const UserHome = props => {
-  const {email} = props;
+import Client from './Client';
 
-  return (
-    <div>
-      <h3>Welcome, {email}</h3>
-    </div>
-  );
-};
+class UserHome extends Component {
+  constructor() {
+    super();
+    this.state = {
+      clients: [],
+      client: {},
+      selectedOption: {},
+    }
+  }
 
-const mapState = state => ({
-    email: state.user.email
-  });
+  componentDidMount() {
+    this.fetchClients();
+  }
 
-export default connect(mapState)(UserHome);
+  async fetchClients() {
```

Comment:
I would abstract these api calls into an api directory, and import relevant methods